package/Dockerfile

```
@@ -329,4 +329,12 @@ VOLUME /var/lib/kubelet
 VOLUME /var/lib/cni
 VOLUME /var/log
 
+LABEL "io.artifacthub.package.logo-url"="https://github.com/rancher/ui/blob/master/public/assets/images/logos/welcome-cow.svg" \
+      "io.artifacthub.package.readme-url"="https://github.com/rancher/rancher/blob/${VERSION}/README.md" \
```

Same comment for the README... it should point to the raw markdown file
```suggestion
      "io.artifacthub.package.readme-url"="https://raw.githubusercontent.com/rancher/rancher/${VERSION}/README.md" \
```

---

@lagartoflojo thanks for pointing this out. I agree we should change the SVG link, so I changed that and aligned the chart accordingly. 

I am not sure we also need that for the `REAME.md`, as the user experience for the average person may be better to see a rendered Markdown instead of its raw form.  PTAL

---

Thanks for updating the SVG link!

Regarding the README: the `io.artifacthub.package.readme-url` label is defined by Artifacthub, and they say:

> io.artifacthub.package.readme-url: url of the readme file (in markdown format) for this package version. Please make sure it points to a raw markdown document, not HTML
> ([source](https://artifacthub.io/docs/topics/repositories/container-images/))

The reason for this is that image catalogs (such as Artifacthub itself and the [SUSE Registry](https://registry.suse.com)) grab that URL and render the markdown in the screen ([example](https://registry.suse.com/repositories/suse-nginx)).

---

Nice, thank you for clarifying. I have updated this accordingly. PTAL